js/Car.js

```diff
@@ -0,0 +1,56 @@
+/**@type{HTMLCanvasElement} */
+
+class Car {
+    constructor(x, y, w, h, ctx, color) {
+        this.x = x;
+        this.y = y;
+
+        this.w = w;
+        this.h = h;
+
+        this.image = new Image();
+        this.color = color;
+        this.ctx = ctx;
+
+        this.speedX = 0;
+        this.speedY = 0;
+    }
+
+    draw() {
+        this.image.src = "/images/car.png";
```

Comment:
The same logic you used here, you could've used to draw the road image instead of having it on CSS. And you could've added it to your game.update function so it would only show up when the start button is clicked